src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final Color color;
+    private final double topLength;
+    private final double bottomLength;
+    private final double height;
+
+    public IsoscelesTrapezoid(Color color,
+                              double topLength,
+                              double bottomLength,
+                              double height) {
+        this.color = color;
```

Comment:
Please create constructor in parent class and call it in children constructors with 'super' keyword.
Applicable for all figures